synapse/handlers/deactivate_account.py

```diff
@@ -129,6 +133,39 @@ def deactivate_account(self, user_id, erase_data, id_server=None):
 
         return identity_server_supports_unbinding
 
+    @defer.inlineCallbacks
+    def _reject_pending_invites_for_user(self, user_id):
+        """Reject pending invites addressed to a given user ID.
+
+        Args:
+            user_id (str): The user ID to reject pending invites for.
+        """
+        user = UserID.from_string(user_id)
+        pending_invites = yield self.store.get_invited_rooms_for_user(user_id)
+
+        logger.info(pending_invites)
```

Comment:
Probably don't want this.

Reply:
Ah, yes, indeed, thanks for spotting that